lib/resources/aws/aws_flow_log.rb

```diff
@@ -0,0 +1,69 @@
+class AwsFlowLog < Inspec.resource(1)
+  name 'aws_flow_log'
+  supports platform: 'aws'
+  desc ''
+  example ''
+
+  include AwsSingularResourceMixin
+
+  def to_s
+    "AWS Flow Log #{@flow_log_id}"
+  end
+
+  attr_reader :log_group_name, :resource_id, :name
+
+  private
+
+  def validate_params(raw_params)
+    validated_params = check_resource_param_names(
+      raw_params: raw_params,
+      allowed_params: [:flow_log_id, :subnet_id, :vpc_id],
+      allowed_scalar_name: :flow_log_id,
+      allowed_scalar_type: String,
+    )
+
+    if validated_params.empty?
+      raise ArgumentError,
+            'aws_flow_log requires a parameter: flow_log_id, subnet_id, vpc_id'
```

Comment:
Might want to add the word 'or' in there; otherwise people might think you have to supply all three.